packages/cli/src/commonlib/azureLoginCI.ts

```diff
@@ -19,11 +17,8 @@ import CLILogProvider from "./log";
 import { LogLevel as LLevel } from "@microsoft/teamsfx-api";
 import * as os from "os";
 import { AzureSpCrypto } from "./cacheAccess";
-import {
-  AzureScopes,
-  ConvertTokenToJson,
-  isV3Enabled,
-} from "@microsoft/teamsfx-core/build/common/tools";
+import { ConvertTokenToJson } from "@microsoft/teamsfx-core/build/common/tools";
+import { InvalidAzureSubscriptionError } from "@microsoft/teamsfx-core";
```

Comment:
Is that still a tip to import from "fx-core/build/..." to avoid any dependency cycle? Or ok to directly import from "fx-core"?